doc/user/walkthrough/backup-operator.md

```diff
@@ -65,6 +100,7 @@ Create EtcdBackup CR:
 ```sh
 sed -e 's|<full-s3-path>|jenkins-testing-operator/etcd.backup|g' \
     -e 's|<aws-secret>|aws|g' \
+    -e 's|<etcd-client-secret>|etcd-client|g' \
```

Comment:
The current `example-etcd-cluster` is an non-TLS cluster.

Reply:
Hi @hongchaodeng. Thanks for the review. Would you suggest that I create another example cluster YAML with TLS, and add a separate sed command for that example?